Warp.WebApp/Properties/launchSettings.json

```diff
@@ -1,14 +1,12 @@
 {
   "profiles": {
-    "https": {
+    "Warp.WebApp": {
       "commandName": "Project",
-      "launchBrowser": false,
+      "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Local"
+        "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "dotnetRunMessages": true,
-      "applicationUrl": "https://localhost:8001;http://localhost:8000"
+      "applicationUrl": "https://localhost:54357;http://localhost:54358"
     }
-  },
-  "$schema": "http://json.schemastore.org/launchsettings.json"
+  }
 }
```

Comment:
We need the default configuration for server runs, so I suggest to add another profile:
```
{
  "profiles": {
    "https": {
      "commandName": "Project",
      "launchBrowser": false,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      },
      "dotnetRunMessages": true,
      "applicationUrl": "https://localhost:8001;http://localhost:8000"
    },
    "kirillta": {
      "commandName": "Project",
      "launchBrowser": false,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Local"
      },
      "dotnetRunMessages": true,
      "applicationUrl": "https://localhost:8001;http://localhost:8000"
    },
    "skirata": {
      "commandName": "Project",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Local"
      },
      "dotnetRunMessages": true,
      "applicationUrl": "https://localhost:54357;http://localhost:54358"
    }
  },
  "$schema": "http://json.schemastore.org/launchsettings.json"
}
```